.circleci/config.yml

```diff
@@ -21,7 +21,8 @@ jobs:
       - run: mix local.hex --force
       - run: mix local.rebar --force
       - run: mix deps.get
-      - run: mix compile --warnings-as-errors
+      - run: mix compile.elixir --warnings-as-errors
+      - run: mix compile
```

Comment:
what's the reason for this change?

Reply:
warnings-as-errors is a flag on `mix compile.elixir` not `mix compile` 


reference: https://hexdocs.pm/mix/Mix.Tasks.Compile.Elixir.html
https://hexdocs.pm/mix/Mix.Tasks.Compile.html#summary

Reply:
btw we have some warnings so build is failing

Reply:
I'll fix it on the long weekend

Reply:
oh no, I promise it's on `mix compile` too, I run it every day

Reply:
ok, reverted